### ISSUE TITLE
Any good chart library for Vue?

### ISSUE THREAD

**Issue by audreychenjj-2019** [susanBuck/e28-fall19#121]
I am learning Plotly Dash to make pretty statistic charts and was planning to use it in my p3. Then I found out Plotly cannot be used in Vue. 

Any recommendation of a good, easy chart library that can be used in Vue?

**Comment by naztronaut** [comment]
I've always use Chart.js for stuff and looks like you can use it in vue: https://www.npmjs.com/package/vue-chartjs

You can search NPM, there are a ton of libraries you can use.

**Comment by gpunzalan18** [comment]
I've used Chartjs and D3. Both of them were good. But Chartjs was a lot easier for me to use!

**Comment by audreychenjj-2019** [comment]
Thanks for the advice.

I tried different things. [Google chart] (https://developers.google.com/chart) is quite easy to use but charts are loaded very slowly.

D3 is too clumsy for plotting a simply line chart.

There is a Plotly wrapper for Vue. Somebody built that. The company Plotly actually said they don't have in-house Vue knowledge. I tested the Plotly wrapper for Vue and found some vulnerability. Thus I ditched it immediately. 

Vue-chartjs is fine and gets the job done. I end up with Vue-chartjs.
